client/src/pages/JoinRoomPage/JoinRoomPage.css

```diff
@@ -0,0 +1,175 @@
+.join_room_page_container {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  background-color: #cfff80;
+}
+
+.join_room_title {
+  font-size: 36px;
+  font-weight: 700;
+  text-align: center;
+}
+
+.join_room_page_panel {
+  display: flex;
+  flex-direction: column;
+
+}
+
+.join_room_inputs_container {
+  display: flex;
+  flex-direction: column;
+  height: 100px;
+  justify-content: space-between;
+  width: 100%;
+  align-items: center;
+  margin-top: 10px;
+}

```

Comment:
There are two identical style blocks in the code with the .join_room_inputs_container class. I recommend removing one of them to avoid duplication.

Reply:
fixed